lib/users/index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const config = require('config');
+const contentTypes = require('punchcard-content-types');
+const userRoutes = require('./routes');
+const fs = require('fs');
+const yaml = require('js-yaml');
+const knex = require('knex')(config.knex);
+
+/*
+  Checks for the existence of a `users` table
+
+  ## TODO
```

Comment:
We have this (and should be in your branch). Can you update? Add to https://github.com/punchcard-cms/punchcard/blob/master/lib/database/schemas.js and you should be GTG